examples/PhasorDynamics/Tiny/ThreeBus/ThreeBus_Basic/Example_ThreeBus_Basic.cpp

```diff
@@ -86,7 +86,7 @@ int main()
 
   auto error_allowed = static_cast<real_type>(1e-4);
 
-  std::cout << "Example 2 version 1\n";
+  std::cout << "Example_ThreeBus_Basic\n";
 
```

Comment:
For the output we could do something like this:
```suggestion
  std::cout << "Example: ThreeBusBasic\n";
```